### DISCUSSION TITLE
Is there a way to turn this into a binding external editing tool?

### DISCUSSION THREAD

**Discussion by DawnTreader** [ghorsey/EdBindings#5] [Ideas]
I really think it is about time for an external bind editor but I am wondering if there is some prohibition against that? Is that something you are working towards here or is this just a viewer?

**Comment by Akallabeth13666** [comment]
> I really think it is about time for an external bind editor but I am wondering if there is some prohibition against that? Is that something you are working towards here or is this just a viewer?

Yea, I think about same thing. It would be very useful when introducing new person to the game. Editing binds file might be hard to handle for some.

**Comment by shodanx2** [comment]
Yes, this is what I was hoping this software was

I wrote a question about this in the elite dangerous subreddit last year

https://old.reddit.com/r/EliteDangerous/comments/fca351/is_there_an_utility_to_make_managing_key_bindings/

Apparently there is still no external keybindings manager for ED

Which is unfortunate since keybinding re-assignements is by far the main reason I am not playing this game anymore.

Every time I think aabout trying this game again, the prospect of hours of tuning key bindings sends me playing something else instead !
